src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs

```diff
@@ -33,5 +33,12 @@ public void VaultCRUDTests()
         {
             TestController.NewInstance.RunPsTest("Test-RecoveryServicesVaultCRUDTests");
         }
+
+        [Fact(Skip = "TODO: only works for live mode")]
```

Comment:
Remove this skip.  This is already marked as a live-only test, now it cannot be run live.